### DISCUSSION TITLE
`using` keyword instead of `//> using`?

### DISCUSSION THREAD

**Discussion by joan38** [scala/scala3#21703] [General Discussion]
This is following [a discussion on Discord about `//> using` in `scala`](https://discord.com/channels/632150470000902164/901153021638082660/1290999693257474058)

Hi,

Sounds like there is no plan of having the top level `using` keyword excluded by the language spec.
Meaning `//>` is necessary in front of `using` with no plan of changing this.

I for any Scala dev that knows the context and history on why the `//>`, it make sense. But for any new comer to Scala it's kind of weird to have to do `//>`.
Most people will not understand the difference between `scalac` and `scala`.
If the compiler (`scalac`) needs to ignore the top level keyword `using`, we should do it instead of relying on some commented (but not really) code.
The `//>` feels like a temporary hack that we should be looking to resolve soon.

Thanks

**Comment by tgodzik** [comment]
We've had a lot of discussions about it and the comment was good enough to work across all Scala versions and not break all the tooling at the same time. 

We could at some point do this only for Scala 3, though it would make it incompatible with Scala 2. 

I think what is missing is someone to push first the discussion and then the SiP through, since the current solution is good enough and hasn't caused a lot of issues.
